src/library/Uno.Material/Styles/Controls/v2/Flyout.xaml

```diff
@@ -17,7 +17,7 @@
 	<x:Double x:Key="MaterialFlyoutMenuToggleCheckGlyphWidth">16</x:Double>
 	<x:Double x:Key="MaterialFlyoutMenuToggleCheckGlyphHeight">16</x:Double>
 	<x:Double x:Key="MaterialFlyoutMenuSeparatorHeight">1</x:Double>
-	<x:Double x:Key="MaterialFlyoutFontSize">14</x:Double>
+	<x:Double x:Key="FlyoutFontSize">14</x:Double>
```

Comment:
not sure why you would rename this? unless this is a special named key by windows fluent style
if that not the case, we can remove this since it is no longer being used here (replaced by LabelLargeFontSize)

Reply:
ideally, you should a validator to check each file after changes